### DISCUSSION TITLE
Error when trying to "Add Untracked Changes" from Git to Github

### DISCUSSION THREAD

**Discussion by Vaslo** [requarks/wiki#5632] [Error / Bug Report]
I am trying to add everything from my instance to Github but as mentioned in a previous post, only changes have been updated.  I have tried to manually add, manually push, etc.  I thought the "Add Untracked Changes" choice would help me but I can getting the following error in the console everytime I try.

![image](https://user-images.githubusercontent.com/7269272/189534151-cf0ae131-75db-4843-a167-5f5bddb46f82.png)

With the following errors/warnings being repeated alot:

![image](https://user-images.githubusercontent.com/7269272/189534202-9d6bc184-2308-482c-8ab7-047ed4212660.png)

I don't know why I'm having such a hard time with Git to Github.  Thanks.

**Comment by simp16** [comment]
Have you figured it out, I am having the same problem…
